CONTRIBUTING.md

```diff
@@ -72,7 +72,8 @@ This is because one may not be emulating bash environment in any way or doing it
 Some hooks use `grep`. The macOS version of `grep` (FreeBSD) differs from GNU `grep`.
 In order to make `grep` and eventually the hooks working one must:
 1. Install `grep` via `brew` (it will not override system's `grep` &mdash; it can be executed as `ggrep`)
-2. Run `brew ls -v grep`; among the other a path like should be found `/opt/homebrew/Cellar/grep/<grep-version>/libexec/gnubin/grep`
+2. Run `brew ls -v grep`; among the other a path like this should be found `/opt/homebrew/Cellar/grep/<grep-version>/libexec/gnubin/grep`
```

Comment:
Ok so IIUC this is hard to get right, pls just extract this update to CONTRIBUTING.md and we're good to go.

Btw have you experienced

Reply:
a bit, yeah. recently I updated this line b/c there was a specific `<grep-version>` given and I was unable to commit (with Git hooks) after updating grep 😆

Reply:
@PawelLipski, what is your proposition `pls just extract this update to CONTRIBUTING.md` about? we are within CONTRIBUTING.md 🤔

Reply:
I should've rather written: just keep the changes to CONTIRIBUTING.md, even if the rest of the changes (to Gradle stuff) is to be discarded 😅